src/mixed-reality/azext_mixed_reality/tests/latest/test_spatial_anchors_account.py

```diff
@@ -4,56 +4,50 @@
+            'storage_account_name': 'storage_account_name',
+            'storage_account_name1': 'storage_account_name1',
         })
 
-        # Create
-        self._assert_spatial_anchors_account_not_exist()
-
-        self._assert_spatial_anchors_account_as_expected('{initial} create -g {rg} -n {name} -l {location}')
-
-        # Read
-        self._assert_spatial_anchors_account_as_expected('{initial} show -g {rg} -n {name}')
-
-        # Primary Key
-        self._assert_spatial_anchors_account_keys_work('primary', 'secondary')
-
-        # Secondary Key
-        self._assert_spatial_anchors_account_keys_work('secondary', 'primary')
-
-        # Delete
-        self.cmd('{initial} delete -g {rg} -n {name}')
-        self._assert_spatial_anchors_account_not_exist()
-
-    def _assert_spatial_anchors_account_not_exist(self):
-        for item in self.cmd('{initial} list -g {rg}').get_output_in_json():
-            self.assertNotEqual(self.kwargs['name'], item['name'])
+        # Create with minimum parameters
+        self.cmd('spatial-anchors-account create -g {rg} -n {account_name}', checks=self.not_exists('tags'))
 
-    def _assert_spatial_anchors_account_as_expected(self, cmd):
-        self.cmd(cmd, checks=[
-            self.check('name', '{name}'),
-            self.check('location', '{location}'),
-        ])
+        # Create with more parameters
+        self.cmd('spatial-anchors-account create -g {rg} -n {account_name1} '
+                 '--storage-account-name {storage_account_name} --tag tag=tag',
+                 checks=[self.exists('tags'),
+                         self.check('storageAccountName', '{storage_account_name}')])
 
-    def _assert_spatial_anchors_account_keys_work(self, changed, unchanged):
-        old = self.cmd('{initial} key show -g {rg} -n {name}').get_output_in_json()
+        self.cmd('spatial-anchors-account update -g {rg} -n {account_name} '
+                 '--storage-account-name {storage_account_name1}',
+                 checks=self.check('storageAccountName', '{storage_account_name1}'))
 
-        self.kwargs['key'] = changed
-        new = self.cmd('{initial} key renew -g {rg} -n {name} -k {key}').get_output_in_json()
+        self.cmd('spatial-anchors-account show -g {rg} -n {account_name}')
+        self.cmd('spatial-anchors-account list -g {rg}', checks=self.check('length(@)', 2))
+        self.cmd('spatial-anchors-account delete -g {rg} -n {account_name1}')
+        self.cmd('spatial-anchors-account list -g {rg}', checks=self.check('length(@)', 1))
 
-        key = unchanged + 'Key'
-        self.assertEqual(old[key], new[key])
+        # key
+        self.cmd('spatial-anchors-account key renew -g {rg} -n {account_name}')
+        self.cmd('spatial-anchors-account key renew -g {rg} -n {account_name} --serial 1')
+        self.cmd('spatial-anchors-account key renew -g {rg} -n {account_name} --serial 2')
+        self.cmd('spatial-anchors-account key show -g {rg} -n {account_name}')
```

Comment:
Could you please add some check for those tests?

Reply:
some cmd have checks. While for other cmd. It is ok to test whether it could run without error.